src/main/java/at/hannibal2/skyhanni/test/TestExportTools.kt

```diff
@@ -52,8 +53,12 @@ object TestExportTools {
 
     @SubscribeEvent
     fun onKeybind(event: GuiScreenEvent.KeyboardInputEvent.Post) {
-        if (!config.copyItemDataCompressed.isKeyHeld() && !config.copyItemData.isKeyHeld()) return
+        if (!config.copyItemDataCompressed.isKeyHeld() && !config.copyItemData.isKeyHeld() && !config.copyChestName.isKeyHeld()) return
```

Comment:
You can remove the extra check in this line and simply move it below the current line 61

Reply:
wouldn't moving it there prevent lines `copyChestName()` from being called? or am i misunderstanding where you want me to place it
<src="https://github.com/hannibal002/SkyHanni/assets/51521765/01d2041b-24bc-4702-a84c-769c8e0a1223">

Reply:
^ that was a screenshot of the original `onKeybind` prior to this PR